pilot/docker/Dockerfile.proxy_init

```diff
@@ -1,8 +1,25 @@
-FROM ubuntu:xenial
+# BASE_DISTRIBUTION is used to switch between the old base distribution and distroless base images
+ARG BASE_DISTRIBUTION=default
+
+# The following section is used as base image if BASE_DISTRIBUTION=default
+FROM ubuntu:xenial as default
 RUN apt-get update && apt-get upgrade -y && apt-get install -y \
     iproute2 \
     iptables \
  && rm -rf /var/lib/apt/lists/*
 
 ADD istio-iptables.sh /usr/local/bin/
+
+# The following section is used as base image if BASE_DISTRIBUTION=distroless
+FROM gcr.io/distroless/cc as distroless
+
+COPY --from=default /sbin/xtables-multi /sbin/iptables* /sbin/ip6tables* /sbin/ip /sbin/
+COPY --from=default /lib/xtables/ /lib/xtables
+COPY --from=default /lib/x86_64-linux-gnu/ /lib/x86_64-linux-gnu
+COPY --from=default /etc/iproute2 /etc/iproute2
+
+ADD istio-iptables /usr/local/bin/istio-iptables.sh
```

Comment:
So we name the binary `.sh`, right? Probably to reuse the `ENTRYPOINT`? I think having a go binary called `istio-tables.sh` is a bit confusing. Is there another possibility? Maybe using a link as `ENTRYPOINT` and letting this link point to the go binary or the script.

Reply:
I changed this